soc/arm/nordic_nrf/nrf53/Kconfig.soc

```diff
@@ -132,6 +132,11 @@ config SOC_DCDC_NRF53X_HV
 	help
 	  Enable nRF53 series System on Chip High Voltage DC/DC converter.
 
+config SOC_CLOCK_128MHZ_NRF53X_APP
+	bool
```

Comment:
This option should be dependent on `CLOCK_CONTROL_NRF` (it can be used only when this driver is enabled) and it should have a prompt, so that it can be configured on the SoC level.